### PR TITLE
[fix]: liquidity quick poll

### PR DESCRIPTION
**Pull request by RDMStreet**

### CODE REVIEW

File: src/modules/ModuleLiquidity/store/remove.ts [hunk 1]
```diff
@@ -76,6 +76,10 @@ function usePrepareSupply(props: {
     usePromiseLog(supplyState, 'liquidity-remove-supply')
     useNotifyOnError(supplyState, notify, 'Supply failed')
 
+    wheneverFulfilled(supplyState, () => {
+      useLiquidityListStore().quickPoll = true
```
Comment (0x009922) [inline src/modules/ModuleLiquidity/store/remove.ts:80]: I am not sure if "using" a store in a watcher callback is a good idea. Hopefully, it works, but in general any composables should be called synchronously during setup.

File: src/modules/ModuleLiquidity/store/add.ts [hunk 2]
```diff
@@ -135,6 +135,7 @@ function usePrepareSupply(props: { tokens: Ref<SupplyTokens | null> }) {
     useNotifyOnError(stateSupply, notify, 'Liquidity addition failed')
     wheneverFulfilled(stateSupply, () => {
       notify({ type: 'ok', description: 'Liquidity addition succeeded!' })
+      useLiquidityListStore().quickPoll = true
```
Comment (0x009922) [inline src/modules/ModuleLiquidity/store/add.ts:138]: Same here.

File: src/modules/ModuleFarming/ModalCard.vue [hunk 3]
```diff
@@ -97,6 +97,7 @@ wheneverFulfilled(operationState, ({ amount, operation }) => {
     emit('unstaked', amount)
   }
 
+  useLiquidityListStore().quickPoll = true
```
Comment (0x009922) [inline src/modules/ModuleFarming/ModalCard.vue:100]: And here.

**Review by 0x009922 — APPROVED** [review]

File: src/modules/ModuleFarming/ModalCard.vue (hunk 3 above)
Comment (RDMStreet) [reply src/modules/ModuleFarming/ModalCard.vue]: Moved outside

File: src/modules/ModuleLiquidity/store/add.ts (hunk 2 above)
Comment (RDMStreet) [reply src/modules/ModuleLiquidity/store/add.ts]: Moved outside

File: src/modules/ModuleLiquidity/store/remove.ts (hunk 1 above)
Comment (RDMStreet) [reply src/modules/ModuleLiquidity/store/remove.ts]: Moved outside

**Pull request merged**